src/app/components/MediaLoader/Metadata/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import filterForBlockType from '#app/lib/utilities/blockHandlers';
+import { AresMediaBlock, AresMediaMetadataBlock, MediaBlock } from '../types';
+
+const getThumbnailUri = (url?: string) => {
+  if (!url) return null;
+
+  let imageUrl = url;
+
+  if (imageUrl.startsWith('http')) {
+    imageUrl = imageUrl.split('/')?.slice(2)?.join('/');
+  }
+
+  return `https://${imageUrl.replace('$recipe', '1024x576')}`;
+};
+
+const getUploadDate = (availableFrom?: string, firstPublished?: string) => {
+  const uploadDate = availableFrom || firstPublished;
+
+  if (!uploadDate) return null;
+
+  return new Date(uploadDate).toISOString();
+};
+
+type Props = {
+  blocks: MediaBlock[];
+  embedURL?: string;
+  embedded?: boolean;
+};
+
+const Metadata = ({ blocks, embedURL, embedded = false }: Props) => {
+  const aresMediaBlock: AresMediaBlock = filterForBlockType(
+    blocks,
+    'aresMedia',
+  );
+
+  if (!aresMediaBlock) return null;
+
+  const { model: metadata }: AresMediaMetadataBlock =
+    filterForBlockType(aresMediaBlock?.model?.blocks, 'aresMediaMetadata') ??
+    {};
```

Comment:
Ah, so this means only aresMediaMetadata is supported. 

Would we need to add additional logic here for our other media pages (on demand TV etc)? (And do we need to add this to the configs, so that buildSettings will also generate the necessary metadata for the media player?

Reply:
I don't think page types other than Articles will need this. The `ArticleMediaPlayer` imports this component, which uses the `Metadata` component to set the JSON+LD: https://github.com/bbc/simorgh/blob/eb13e8820661c929cd4cb7c3b2ebb0e0d65cc6a4/src/app/legacy/containers/MediaPlayer/index.jsx#L148

Other page types use `AVPlayer` directly: https://github.com/bbc/simorgh/blob/eb13e8820661c929cd4cb7c3b2ebb0e0d65cc6a4/src/app/legacy/containers/AVPlayer/index.jsx with no `Metadata` component.

But this is a good shout. I can restrict this component to only render where appropriate.

Reply:
We're in the process of checking that the new media loader works as expected on the on demand TV page (see #11928)

Reply:
I've added checks to the new `Metadata` component to return nothing if the page type is considered 'unsupported'. So this component is quite opinionated towards `aresMedia`, article assets atm.

Reply:
That's OK - we can see if we need this for the other media page types, and can refactor then if needed